docs/source/install/ansible.rst

```diff
@@ -88,6 +91,14 @@ Below is more advanced example to customize |st2| deployment:
         - name: Install st2web
           role: st2web
 
+        - name: Install st2chatops with "slack" hubot adapter
+          role: st2chatops
+          vars:
+            st2chatops_version: latest
+            st2chatops_st2_api_key: CHANGE-ME-PLEASE # This can be generated using "st2 apikey create -k"
+            st2chatops_hubot_adapter: slack
+            st2chatops_config: {"HUBOT_SLACK_TOKEN":"xoxb-CHANGE-ME-PLEASE"}
```

Comment:
You are right, the assertion in the play was not correct: https://github.com/StackStorm/ansible-st2/pull/129/commits/2cabe7ee42be6570014448881a2853fd4fe9cb66#diff-b9eaf1c4002cf73f79f7506ed1c4f718L13.